number_test.go

```diff
@@ -334,6 +336,148 @@ func TestNumber_InDelta(t *testing.T) {
 	}
 }
 
+func TestNumber_InDeltaRelative(t *testing.T) {
+	cases := map[string]struct {
```

Comment:
nit: could you use slices of structs instead of map of structs?

See https://github.com/gavv/httpexpect/issues/324